HELP.md

```diff
@@ -1,8 +1,8 @@
 # Module for Extron SMP 351
 
-> this module connects via a telnet connection.
+This module connects to the Extron H.264 Streaming Media Processor (SMP) 351 and allows you to login, send commands, and recieve feedback based on the status of the device.
```

Comment:
Leave "> " in, It's formatting, purely cosmetic, I think it looks nicer.

Reply:
Got it, re-added.